deploy/examples/create-external-cluster-resources.py

```diff
@@ -1017,6 +1018,7 @@ def get_defaultUser_caps_and_entity(self):
             "mon": "allow r, allow command quorum_status, allow command version",
             "mgr": "allow command config",
             "osd": f"profile rbd-read-only, allow rwx pool={self._arg_parser.rgw_pool_prefix}.rgw.meta, allow r pool=.rgw.root, allow rw pool={self._arg_parser.rgw_pool_prefix}.rgw.control, allow rx pool={self._arg_parser.rgw_pool_prefix}.rgw.log, allow x pool={self._arg_parser.rgw_pool_prefix}.rgw.buckets.index",
+            "mds": "allow *",
```

Comment:
can we just give  `    "mds": "allow rw",`

Reply:
Taking with cephfs people, they said it requires `*`

Reply:
@subhamkrai can we try with "mds": "allow rw",??

Reply:
And it seems only cephfs provisioner require it, have you checked that healthcheker also require it

And please add a comment providing permission for mds node fencing

Reply:
yes both requires the caps

Reply:
caps are added in cephfs/mds entity